src/strategy/paladin/PaladinActions.cpp

```diff
@@ -12,13 +12,23 @@
 #include "PlayerbotFactory.h"
 #include "Playerbots.h"
 #include "SharedDefines.h"
+#include "../../../../../src/server/scripts/Spells/spell_generic.cpp"
 
-inline std::string const GetActualBlessingOfMight(Unit* target)
+inline std::string const GetActualBlessingOfMight(Unit* target, PlayerbotAI* botAI, Player* bot)
 {
-    if (!target->ToPlayer())
+    Player* targetPlayer = target->ToPlayer();
+
+    if (!targetPlayer)
     {
         return "blessing of might";
     }
+
+    if (targetPlayer->HasTankSpec() && !targetPlayer->HasAura(SPELL_BLESSING_OF_SANCTUARY) &&
```

Comment:
This is why it doesn't work properly. If the user has the aura he reverts to might (when targeting a protection paladin). It shouldn't check if the target has the aura because it doesn't for the other spells.